go/ct/common/opcodes_test.go

```diff
@@ -65,3 +65,51 @@ func TestOpCode_CanBePrinted(t *testing.T) {
 		}
 	}
 }
+
+func TestOpCode_NumberOfOpCodes(t *testing.T) {
+	istanbulOpCodes := []OpCode{
+		0x00, 0x01, 0x02, 0x03, 0x04, 0x05, 0x06, 0x07, 0x08, 0x09, 0x0A, 0x0B,
```

Comment:
It would be great to have the actual OpCode constants here, instead of their values.

So, for instance `[]OpCode{STOP, ADD, MUL}`.